qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -35,21 +35,24 @@ def __init__(self, provider: qss.SuperstaqProvider, target: str) -> None:
             target: A string containing the name of a target backend.
         """
         self._provider = provider
+
+        target_info = self._provider._client.target_info(target)["target_info"]
         self.configuration_dict = {
             "backend_name": target,
             "backend_version": "n/a",
-            "n_qubits": -1,
-            "basis_gates": None,
+            "n_qubits": target_info["num_qubits"],
+            "basis_gates": target_info["native_gate_set"],
             "gates": [],
             "local": False,
             "simulator": False,
             "conditional": False,
             "open_pulse": False,
```

Comment:
Should something like this also be added?
```suggestion
            "conditional": True if target_info["conditional"] else False ,
            "open_pulse": True if target_info["open_pulse"] else False ,
```

Reply:
I think these checks are already taken care of when the target info is initialized in the client

Reply:
Oh i see what's going on

@bharat-thotakura  So the reason to do something like this is when the keys the target info endpoint if returning is different from the ones that the configuration dict wants. For example `"n_qubits": target_info["num_qubits"],`

In the case of your suggestions, `target_info` endpoint already returns the `condition` and `open_pulse` keys which are then updated in `self.configuration_dict.update(target_info)`